OutOfSchool/OutOfSchool.DataAccess/Models/Tag.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace OutOfSchool.Services.Models;
+public class Tag : IKeyedEntity<long>
+{
+    public long Id { get; set; }
+
+    public string Name { get; set; } = string.Empty;
```

Comment:
why do you need default values ?

Reply:
Assuming this (1)entity can not be simply added by a regular user, it doesn't require the default value, I have simply followed the way it was implemented with other entities sharing this(1) trait (for instance SocialGroup). Anyways, even though it doesn't affect the performance as it is not a frequent event that this entity's instance is added, if you insist, I can sure get rid of this.